electron/store.js

```diff
@@ -35,19 +35,21 @@ const setupStoreIpc = (ipcMain, mainWindow) => {
    * Initially the store was setup with only trader agent settings.
    * The following code migrates the old store to the new store schema.
    */
-  const traderAgent = {
-    ...(store.get('trader') || {}),
-    isInitialFunded:
-      store.get('isInitialFunded_trader') ||
-      store.get('isInitialFunded') ||
-      false,
```

Comment:
this makes it set to false for all users who have isInitialFunded / isInitialFunded_trader removed, even though trader. isInitialFunded could have been true (we simply did `store.set('trader', traderAgent);` below)